portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/IndividualsAttributeBreakdown.testcase

```diff
@@ -91,42 +87,34 @@ definition {
 			ACProperties.switchProperty(propertyName = "${assignedPropertyName}");
 		}
 
-		task ("Add a CSV Data Source") {
-			ACSettings.addCSVDataSource(
-				dataSourceName = "CSV Data Source",
-				sourceFileName = "jobTitle.csv");
-		}
-
-		task ("Exit Settings") {
-			ACNavigation.exitSettings();
-		}
-
```

Comment:
Looking at this test before, I am confused why it ever added a CSV file in the first place 🤔

Reply:
I was confused too. I think he thought the graph only worked with CSV. This test was from the automation task force, not everyone knows how things worked.